bumblebee_status/modules/core/xrandr.py

```diff
@@ -40,40 +40,50 @@ class Module(core.module.Module):
     def __init__(self, config, theme):
         super().__init__(config, theme, [])
 
+        self._exclude = tuple(filter(len, self.parameter("exclude", "").split(",")))
```

Comment:
instead of manually splitting here, you could use the method util.format.aslist(), which gives uniform list formatting (incidentially, as comma-separated list)